operator/src/main/java/oracle/kubernetes/operator/http/rest/model/AdmissionResponse.java

```diff
@@ -29,6 +31,13 @@ public class AdmissionResponse {
   @Expose
```

Comment:
These fields should all be private. We have accessors for any code outside the class that needs them.